### ISSUE TITLE
FBRetainCycleDetector Crashing

### ISSUE THREAD

**Issue by vsravan** [facebook/FBRetainCycleDetector#60]
**I have same issue**.
I have Integrated FBMemory Profiler In our App till now it is good it shows Retain Cycles. Recently we have update to Swift 4.0 compiling  Our App from Xcode 9.2  With FBMemoryProfiler when I'm Clicking on RetainCycle Button App is Crashing.

**Comment by Megra** [comment]
Can you post the full logs showing the symbolication stacktrace of the threads?

**Comment by vsravan** [comment]
Right now I'm not able install the FBMemory Profiler Pod file. it is showing **[!] Unable to find a specification for `FBMemory Profiler`**

**Comment by zhangxqq** [comment]
I have also crashed in ios8.3, 
<img width="1349" alt="screen shot 2018-03-08 at 2 49 18 pm" src="https://user-images.githubusercontent.com/956549/37137247-f60df4b6-22df-11e8-8d76-371e9d204d7a.png">

**Comment by Megra** [comment]
Yeah the crash on [top nextObject] and FBWrapObjectGraphElement is something known but we haven't managed to fix it. Feel free to try finding a fix for this :)

**Comment by Yincongxiao** [comment]
> @Megra <img alt="screen shot 2018-03-07 at 4 31 20 pm" width="1642" src="https://user-images.githubusercontent.com/30058289/37126247-02fc4cb0-2225-11e8-9d62-6af478f24888.png">

i have same issue when enumerate `NSMapTable`.
